kvbc/include/merkle_tree_block.h

```diff
@@ -12,15 +12,23 @@
 
 namespace concord::kvbc::v2MerkleTree::block::detail {
 
-// Creates a block that includes a set of key/values. The passed parentDigest buffer must be of size BLOCK_DIGEST_SIZE
-// bytes.
+// Creates a block that adds a set of key/values.
 RawBlock create(const SetOfKeyValuePairs &updates,
                 const BlockDigest &parentDigest,
                 const sparse_merkle::Hash &stateHash);
 
+// Creates a block that adds a set of key/values and a deletes a set of keys.
```

Comment:
The latter is probably "a set of keys to delete"

Reply:
Thanks, fixing it!